.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence.
```

Comment:
do you want to update the comment?
Unless a later match takes precedence. - this doesn;t make sense now.

Reply:
Removed that part.